fixbaserom.py

```diff
@@ -68,11 +73,19 @@ def get_str_hash(byte_array):
 print("Patching header...")
 fileContent[0x3E] = 0x50
 
+for i in range(0x35CF000, len(fileContent)):
+    fileContent[i] = 0xFF
+
 # Check to see if the ROM is a "vanilla" Debug ROM
 str_hash = get_str_hash(bytearray(fileContent))
 if str_hash != "f0b7f35375f9cc8ca1b2d59d78e35405":
     print("Error: Expected a hash of f0b7f35375f9cc8ca1b2d59d78e35405 but got " + str_hash + ". " +
           "The baserom has probably been tampered, find a new one")
+
+    if str_hash == "32fe2770c0f9b1a9cd2a4d449348c1cb":
+        print("The provided baserom is a rom which has been edited with ZeldaEdit and is not suitable for use with decomp. " +
+              "Find a new one.")
```

Comment:
Isn't this just redundant with the message we already provide?

Reply:
It is, but imo acknowledging an unsupported ROM is better than just saying the ROM isn't supported (I may even want to add a check for the oot 1.0 n64 rom and other versions...)
And it also helps make a user aware there are several variants of the rom

It's common for the average user to ignore an error message, regardless of how nice and friendly it is. Hopefully duplicating the "find a new one" statement gets the message across better
It's not like this is cluttering the `make setup -j` output by producing 3 times as much prints